File: aoc2015/day1/part1.js
const fs = require('fs').promises;


async function main() {
  const file = await fs.readFile('./input.txt', 'utf8');
  const fileContents = file.trim().split('');


  let floor = 0;

  fileContents.forEach((item) => {
    if (item === '(') {
      floor += 1;
    }
    else if (item === ')') {
      floor -= 1;
    }
  });
  console.log(floor);
}



try {
  main();
}
catch (err) {
  console.log(err);
}

File: aoc2018/src/day1/part1.go
package main

import (
	"fmt"
	"log"
	"os"
	"strconv"
	"strings"
)

func freqCount(input string) int {
	var sum int
	for _, line := range strings.Split(input, "\n") {
		temp, err := strconv.Atoi(line)
		if err != nil {
			log.Fatal(err)
		}
		sum += temp
	}
	return sum
}

func main() {
	content, err := os.ReadFile("./input.txt")
	if err != nil {
		log.Fatal(err)
	}
	strcontent := string(content)
	fmt.Println(freqCount(strings.TrimRight(strcontent, "\n")))

}
